src/screens/HtmlScreen.js

```diff
@@ -36,14 +36,25 @@ export class HtmlScreen extends React.PureComponent {
     const { navigation } = this.props;
     const query = navigation.getParam('query', '');
     const queryVariables = navigation.getParam('queryVariables', '');
-    const title = navigation.getParam('title', '');
-    const rootRouteName = navigation.getParam('rootRouteName', '');
-    const subQuery = navigation.getParam('subQuery', '');
 
     if (!query || !queryVariables || !queryVariables.name) return null;
 
+    const title = navigation.getParam('title', '');
+    const rootRouteName = navigation.getParam('rootRouteName', '');
+    const subQuery = navigation.getParam('subQuery', '');
     const isConnected = this.context.isConnected;
     const fetchPolicy = graphqlFetchPolicy(isConnected);
+    // action to open source urls
+    const openWebScreen = (webUrl) => {
+      return navigation.navigate({
+        routeName: 'Web',
```

Comment:
Here is a mistake. A condition is necessary to decide, whether to navigate to `Web` or a given `routeName` from `subQuery`. I fixed it with https://github.com/ikuseiGmbH/smart-village-app-app/commit/1baffbb3baa72c783b1bbc5f5bef32cb41859b56